code/modules/mob/language/station_vr.dm

```diff
@@ -88,23 +88,26 @@
 	desc = "Shadekin seem to always know what the others are thinking. This is probably why."
 	speech_verb = "mars"
 	ask_verb = "mars"
-	exclaim_verb = "mars"
+	exclaim_verb = "MARS"
 	colour = "changeling"
 	key = "m"
 	machine_understands = FALSE
 	flags = WHITELISTED | HIVEMIND
 
-/datum/language/human/slavic
+/datum/language/slavic
 	name = LANGUAGE_SLAVIC
 	desc = "The official language of the Independent Colonial Confederation of Gilgamesh, originally established in 2122 by the short-lived United Slavic Confederation on Earth."
-	colour = "solcom"
+	speech_verb = "govorit" // All Russian for "says" "asks" and "shouts". Mostly here as a joke.
+	ask_verb = "voprosy"
+	exclaim_verb = "krichit"
+	whisper_verb = "shepchet"
+	colour = "attack" // Red for slav
 	key = "r"
```

Comment:
So this is the problem?

Reply:
Yes, another language has the same key, when two languages have the same key, the code that is responsible for Gibberish gets confused and converts all attempts to use that language into gibberish.

Reply:
Then what other language uses the same key? Furthermore, I _thought_ keys could be selected in preference setup.

Reply:
You can set the key markers (like using &8) using setup iirc